nextflow_schema.json

```diff
@@ -300,6 +300,11 @@
             "description": "",
             "default": "",
             "properties": {
+                "delly_scatter_size": {
+                    "type": "integer",
+                    "default": 1000000,
```

Comment:
Here are the lengths of the chromosomes for the bigger test crams. 
>chr14  LN:107043718  
>chr16  LN:90338345 
>chrX  	LN:156040895
I would suggest for now to choose a default size so we end up with max 50 chunks for Delly. Let's say 3,000,000-4,000,000?